sample.env

```diff
@@ -0,0 +1,8 @@
+SEC_CRYPTO_SERVICE_BASE_URI=
+#The REST endpoint path of the xternal service if `sec.useHsm=false`

```

Comment:
Should this be changed to 'external' instead of 'xternal'?

Reply:
Yep, good catch! Fixed.